src/routers/Gateway.sol

```diff
@@ -28,13 +28,22 @@ interface RouterLike {
     function send(bytes memory message) external;
 }
 
+interface AuthLike {
+    function rely(address usr) external;
+}
+
 contract ConnectorGateway {
     using TypedMemView for bytes;
     // why bytes29? - https://github.com/summa-tx/memview-sol#why-bytes29
     using TypedMemView for bytes29;
     using ConnectorMessages for bytes29;
 
     mapping(address => uint256) public wards;
+    mapping(address => uint256) public relySchedule;
+    uint256 public immutable SHORT_SCHEDULE_WAIT;
+    uint256 public immutable LONG_SCHEDULE_WAIT;
+    uint256 public immutable GRACE_PERIOD;
```

Comment:
Two small comments: i think it would be good to add what the grace period is for, and I believe the standard is to use camel casing still for immutable variables, and only use all uppercase for constants.

```suggestion
    uint256 public immutable shortScheduleDelay;
    uint256 public immutable longScheduleDelay;
    uint256 public immutable scheduleGracePeriod;
```